pvm/programs/inst_load_imm_and_jump_indirect_different_regs_without_offset_ok.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "inst_load_imm_and_jump_indirect_different_regs_without_offset_ok",
+  "initial-regs": [
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0
+  ],
+  "initial-pc": 0,
+  "initial-page-map": [],
+  "initial-memory": [],
+  "initial-gas": 10000,
+  "program": [
+    1,
+    1,
+    19,
+    9,
+    51,
+    7,
+    2,
+    180,
+    120,
+    2,
+    210,
+    4,
+    0,
```

Comment:
@koute I would like your feedback on the following.

The opcode bitmask (`k`) tells us this byte (L35) with value 0 is an instruction byte (`trap` or `0`).
At the same time it seems to be an argument byte for the previous instruction (`load_imm_jump_ind` or `180`). Specifically the last immediate argument [`V_Y`](https://graypaper.fluffylabs.dev/#/5f542d7/29bd0129ca01).

It cannot be both an argument byte and an instruction byte. So I suspect something is wrong with this testvector. Please advise.